runtime/development/src/lib.rs

```diff
@@ -715,11 +715,15 @@ impl pallet_claims::Config for Runtime {
 	type WeightInfo = ();
 }
 
+// Parameterize pools pallet
```

Comment:
This one doesn't say pool config parameters.
I would suggest `Pool config parameters` maybe